test/integration/policy_info_metric_test.go

```diff
@@ -155,19 +207,28 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test policy_governance_info metric
 		By("Checking the policy metric")
 		policyLabel := `policy="` + noncompliantPolicyName + `"`
 		Eventually(func() interface{} {
-			resp, _, err := common.GetWithToken(propagatorMetricsURL, strings.TrimSpace(metricToken))
+			resp, _, err := common.GetWithToken(
+				context.Background(),
+				propagatorMetricsURL,
+				strings.TrimSpace(metricToken),
+			)
 			if err != nil {
 				return err
 			}
+
 			return resp
 		}, defaultTimeoutSeconds, 1).Should(common.MatchMetricValue(metricName, policyLabel, "1"))
 	})
 	It("Cleans up", func() {
-		common.OcHub("delete", "-f", compliantPolicyYaml, "-n", userNamespace)
-		common.OcHub("delete", "-f", noncompliantPolicyYaml, "-n", userNamespace)
-		common.OcHub("delete", "route", "-n", ocmNS, "-l", propagatorMetricsSelector)
-		common.OcHub("delete", "clusterrolebinding", roleBindingName)
-		common.OcHub("delete", "serviceaccount", saName, "-n", userNamespace)
-		common.OcHub("delete", "namespace", "policy-metric-test-compliant")
```

Comment:
Here no longer need to delete namespace here after https://github.com/stolostron/governance-policy-framework/blob/main/test/integration/integration_suite_test.go#L90

err is `namespaces \"policy-metric-test-compliant\" not found` rather than nil